src/tools/pub/troubleshoot.md

```diff
@@ -8,28 +8,30 @@ description: "Common gotchas you might run into when using pub."
 
 You receive the following error when running `pub publish`:
 
-{% prettify none %}
+```terminal
+$ pub publish ...
```

Comment:
I don't like adding the commands to these, because the emphasis should be on the error, not the command. (also, the preceding text says "the following error", so it seems wrong to include the command)

Reply:
Ok, I'll take them out.

Reply:
I've removed the commands, reverting to what was there on the page before my edits. But I've managed to find a way to keep the "terminal output" styling (which is what I was looking for before). E.g.:

> ![image](https://user-images.githubusercontent.com/4140793/65331953-c004fc00-db8b-11e9-9d76-29826320c107.png)

as compared to what we had before:

> ![image](https://user-images.githubusercontent.com/4140793/65331980-d448f900-db8b-11e9-8395-8ce9dd45ebe8.png)

WDYT?

Reply:
LGTM